oauth2.py

```diff
@@ -354,6 +356,7 @@ def create_tweet_payload(text, in_reply_to_tweet_id=None):
 
         tweets = chunk_tweet(message)
         tweet_id = None
+        status = "failed"
```

Comment:
No need to declare and assign the `status` variable here, since we can pass the values directly to the `store_publication` function argument.